### PR TITLE
Another round of UI finesse fixes

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: #9737
Resolves: #9223 
Resolves: #9690 

NOTE: I couldn't really test the Learn page because I have no YouTube API key, so I see no thumbnails at all.

### CODE REVIEW

File: src/notation/qml/MuseScore/NotationScene/ConcertPitchControl.qml [hunk 1]
```diff
@@ -33,10 +33,13 @@ FlatButton {
 
     orientation: Qt.Horizontal
     transparent: true
+    margins: 4
 
     contentItem: Row {
         spacing: 6
 
+        width: implicitWidth + 4 // some extra right padding
```
Comment (RomanPudashkin) [inline src/notation/qml/MuseScore/NotationScene/ConcertPitchControl.qml:41]: As far as I understand, you can rewrite your code like this:

width: implicitWidth + root.margins

File: src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml [hunk 2]
```diff
@@ -228,7 +228,7 @@ Slider {
                 ctx.textAlign = "end"
             }
 
-            ctx.clearRect(0, 0, bgCanvas.width, bgCanvas.height)
+            ctx.clearRect(0, 0, bgCanvas.height, bgCanvas.width)
```
Comment (RomanPudashkin) [inline src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml:231]: Could you please explain what is the purpose of this change?

File: src/notation/qml/MuseScore/NotationScene/ZoomControl.qml [hunk 3]
```diff
@@ -49,6 +49,8 @@ RowLayout {
         icon: IconCode.ZOOM_IN
         iconFont: ui.theme.toolbarIconsFont
 
+        width: 28
+        height: 28
```
Comment (RomanPudashkin) [inline src/notation/qml/MuseScore/NotationScene/ZoomControl.qml:53]: Better to use bindings in such situations:

width: 28
height: width

File: src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml (hunk 2 above)
Comment (cbjeukendrup) [reply src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml:231]: It's because we're in a rotated world here, so width and height are reversed. 
(In the lines before this, the `ctx` is rotated.)

**Comment by jessjwilliamson** [comment]
Thumbnails have disappeared in Learn page!

**Comment by cbjeukendrup** [comment]
> Thumbnails have disappeared in Learn page!

You mean the Learn page is completely empty? That's right, because test builds for pull requests don't have access to the Youtube API key, so therefore they can't download the playlist from Youtube. So we'll probably see the result only in nightlies, when this PR is merged.

**Comment by jessjwilliamson** [comment]
> > Thumbnails have disappeared in Learn page!
> 
> You mean the Learn page is completely empty? That's right, because test builds for pull requests don't have access to the Youtube API key, so therefore they can't download the playlist from Youtube. So we'll probably see the result only in nightlies, when this PR is merged.

Ah! that's fine :) I can see changes on the plugins and that looks good!

File: src/notation/qml/MuseScore/NotationScene/ZoomControl.qml (hunk 3 above)
Comment (cbjeukendrup) [reply src/notation/qml/MuseScore/NotationScene/ZoomControl.qml:53]: Done!

File: src/notation/qml/MuseScore/NotationScene/ConcertPitchControl.qml (hunk 1 above)
Comment (cbjeukendrup) [reply src/notation/qml/MuseScore/NotationScene/ConcertPitchControl.qml:41]: Semantically, not really, I think. That extra padding is not necessarily meant to be the same as the normal margin.

**Review by RomanPudashkin — APPROVED** [review]
